crates/rerun/src/web_viewer.rs

```diff
@@ -100,3 +98,43 @@ impl crate::sink::LogSink for RemoteViewerServer {
 pub fn new_sink(open_browser: bool) -> Box<dyn crate::sink::LogSink> {
     Box::new(RemoteViewerServer::new(open_browser))
 }
+
+/// Hosts the rerun web assets only
```

Comment:
What do you mean by "web assets only" - as opposed to what?

Reply:
As opposed to also running the websocket server.